### PR TITLE
docs: fix missing 'Product' Interface import

### PR DESCRIPTION
**Pull request by Nabeelhassan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/angular/angular/issues/42771

## What is the new behavior?
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/examples/getting-started/src/app/cart.service.ts [hunk 1]
```diff
@@ -2,13 +2,14 @@
 // #docregion import-http
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+// #docregion props
 import { Product } from './products';
 // #enddocregion import-http
```
Comment (gkalpak) [inline aio/content/examples/getting-started/src/app/cart.service.ts:7]: Since we only want to show the relevant bits in the `props` docregion, I think it would be better to end the `props` docregion here (and revert the change on line 12):

```suggestion
// #enddocregion import-http, props
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @Nabeelhassan!

I've left a small suggestion. Also, can you please update the commit message to more closely follow our [commit message guidelines](https://github.com/angular/angular/blob/655507012b7b59c83e27429349ce3ac10608a82d/CONTRIBUTING.md#commit) (for example, include a reference to the fixed issue):

> docs: include \`Product\` interface import in \`props\` docregion
>
> Fixes #42771

**Comment by Nabeelhassan** [comment]
closing pull request in favour of https://github.com/angular/angular/pull/42701 :(
